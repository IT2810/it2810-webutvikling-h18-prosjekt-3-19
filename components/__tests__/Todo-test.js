import React from "react";
import Todo from "../Todo";
import renderer from "react-test-renderer";

test("Todo renders correctly", () => {
  const tree = renderer.create(<Todo val="" />);
  expect(tree.toJSON()).toMatchSnapshot();
});
