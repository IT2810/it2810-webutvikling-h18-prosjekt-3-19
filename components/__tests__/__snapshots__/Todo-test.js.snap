// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#ededed",
      "borderBottomWidth": 2,
      "padding": 20,
      "paddingRight": 100,
      "position": "relative",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "borderLeftColor": "#E91E63",
        "borderLeftWidth": 10,
        "paddingLeft": 20,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "borderLeftColor": "#E91E63",
        "borderLeftWidth": 10,
        "paddingLeft": 20,
      }
    }
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#2980b9",
        "bottom": 10,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 10,
        "position": "absolute",
        "right": 10,
        "top": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
        }
      }
    >
      D
    </Text>
  </View>
</View>
`;
